### PR TITLE
minimum requirements met

### PR DESCRIPTION
**Pull request by MaxGunter99**
I CANT BELIEVE I SPELLED MY OWN NAME WRONG

### CODE REVIEW

File: instagram/instagram/src/Components/dataLayout.js [hunk 1]
```diff
@@ -1,14 +1,31 @@
...
+                        <img className='profile' src={dummyData.thumbnailUrl} alt={dummyData.username} />
+                        <p><strong>{dummyData.username}</strong></p></div>
+                    <img className='picture' src={dummyData.imageUrl} alt={dummyData.username} />
+
+                    {/* likes */}
+                    <p className='likes'>{dummyData.likes} Likes</p>
+
+                    {/* comments */}
+                    <div className='comments'>
+                        {dummyData.comments &&
```
Comment (nomadic4life) [inline instagram/instagram/src/Components/dataLayout.js]: this condition here is missing the `.length`  should be `dummyData.comments.length` but I am going to assume that that first condition is never being met, or always evaluating to true. so it is always going to revert to the second condition whether or not the comments array is empty or not.

File: instagram/instagram/src/App.js [hunk 2]
```diff
@@ -1,22 +1,29 @@
...
+    this.setState({
+      dummyData: Info
+    });
+  }
+
   render() {
+    
     return (
       <div className="App">
         <PostContainer dummyData={this.state.dummyData} />
```
Comment (nomadic4life) [inline instagram/instagram/src/App.js]: Where is your search component. I wonder if it is in your PostContainer?

File: instagram/instagram/src/Components/App.js [hunk 3]
```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+
```
Comment (nomadic4life) [inline instagram/instagram/src/Components/App.js]: 2 App components? one in Components and another in src? why do you have it set up this way?

File: instagram/instagram/src/Components/App.js [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+          handleAddTodo={this.addTodo}
+
+          handleClearTodos={this.clearCompletedTodos}
+        />
+      </div>
+    );
+  }
+}
+
+export default App;
```
Comment (nomadic4life) [inline instagram/instagram/src/Components/App.js]: ohhh is this copied from todo? I think I get what you are trying to do here.

File: instagram/instagram/src/Components/Comments/commentapp.js [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+      comments: [],
+      comment: ''
+    };
+}
+
+addComment = event => {
+  event.preventDefault();
+  const newComment = {task: this.state.comment, completed: false, id: Date.now() };
+  this.setState({
+    comments: [...this.state.comments, newComment],
```
Comment (nomadic4life) [inline instagram/instagram/src/Components/Comments/commentapp.js]: I would have the comments updated in the `App.js` I think we got that set up yesterday. if we did we can ignore this comment

**Comment by nomadic4life** [comment]
nicely done with your commits. everything looks good. just a few nitpicks here and there

**Comment by MaxGunter99** [comment]
I think I get what your saying, Im still working out the search bar and
its not able to sort through my post container because its not connected,
its just the search bar, this is connecting some dots for me

**Pull request merged**